src/containers/LoginForm.js

```diff
@@ -0,0 +1,66 @@
+// @flow
```

Comment:
we can:
1) make forms manually, but then in every form, we have to handle much dirty manually like dirty , pristine etc field states.
2) use some libs. There aren't too many though: https://github.com/foxhound87/mobx-react-form and https://github.com/formstate/formstate . I don't like both APIs, but `mobx-react-form` seems more powerful in general and seems 1)(writing every form manually) not an option(we finally will end up with our own lib, heh)...so I think `mobx-react-form`?